### PR TITLE
tests for defined_resource_types

### PR DESCRIPTION
**Pull request by kjhenner**
LEARNVM-501 #resolved #time 2h 30m

### CODE REVIEW

File: tests/defined_resource_types_spec.rb [hunk 1]
```diff
@@ -1,90 +1,65 @@
...
-  it "Apply your test manifest again to create your user's public_html directory and index.html document" do
-    file('/home/shelob/public_html')
-      .should be_directory
-    file('/home/shelob/public_html/index.html')
+  it 'Trigger a Puppet run on pasture-app-small.puppet.vm to enforce your changes' do
+    file('/home/bert/authorized_keys')
       .should be_file
   end
 end
+~
```
Comment (greglarkin) [inline tests/defined_resource_types_spec.rb]: Is this a spurious character?

Comment (kjhenner) [inline tests/defined_resource_types_spec.rb]: @greglarkin I'm not sure what character you're referring to

Comment (greglarkin) [inline tests/defined_resource_types_spec.rb]: I looks like there's a tilde (`~`) on line 64 above

**Comment by kjhenner** [comment]
@greglarkin Should be set now

**Pull request merged**